.golangci.yml

```diff
@@ -27,7 +16,11 @@ linters-settings:
     disabled-checks:
       - singleCaseSwitch
   gci:
-    local-prefixes: github.com/kumahq/kuma
+    sections:
+    - standard
+    - default
+    - prefix(github.com/kumahq/kuma)
```

Comment:
master has [kuma-net](https://github.com/kumahq/kuma/blob/abf0346477a9e8d4dba0f0c74a1c06fd7d782e83/.golangci.yml#L22) here as well, I guess we don't need it because there was no kuma-net then, right?

Reply:
Yes, that correct, there were no kuma-net